test/lib/ex_debug_toolbar/phoenix_test.exs

```diff
@@ -40,9 +22,15 @@ defmodule ExDebugToolbar.PhoenixTest do
     assert make_request().assigns[:called?] == true
   end
 
+  test "it tracks execution time of all following plugs in pipeline" do
+    make_request timeout: 100
+    assert {:ok, request} = get_request()
+    assert Timeline.duration(request.data.timeline) > 70 * 1000 # not sure why
```

Comment:
duration is in microseconds, sleep timer is in milliseconds. 
but looking at test, duration/1 seems redundant now. 
what I couldn't figure out is why duration was almost always less than sleep time